platforms/chibios/chibios_config.h

```diff
@@ -28,6 +28,11 @@
 #    define USE_GPIOV1
 #    define PAL_OUTPUT_TYPE_OPENDRAIN _Static_assert(0, "RP2040 has no Open Drain GPIO configuration, setting this is not possible");
 
+#    define BACKLIGHT_PAL_MODE (PAL_MODE_ALTERNATE_PWM | PAL_RP_PAD_DRIVE12 | PAL_RP_GPIO_OE)
+#    define BACKLIGHT_PWM_COUNTER_FREQUENCY 490196
```

Comment:
Maybe write this as `125000000/255` instead of using a strange number of unknown origin?

Although the datasheet says that dividing by 256 should be supported too, so this might be `125000000/256` = `488281` (with the `0.25` remainder truncated). Or maybe just use `1000000`, which should be obtainable precisely even if someone decides to run the chip on another frequency than the maximum 125 MHz, and would give a slightly better brightness resolution (3906 steps at 256 Hz output frequency instead of 1914).